README.md

```diff
@@ -48,76 +31,68 @@ You need to have the following installed:
 If you want to benchmark Java, you will also need:
   * A Java SDK 7 (`openjdk-7-jdk` package in Debian)
 
-Note that to use pinning on Linux, `cset shield` must be in a working state.
-Some Linux distributions have been known to ship with this functionality
-broken. See the `cset` tutorial for information on how to test `cset shield`:
-https://rt.wiki.kernel.org/index.php/Cpuset_Management_Utility/tutorial
+You will need either `sudo` (on Linux) or `doas` (OpenBSD) to be setup.
 
-### Kernel arguments
 
-If you are using a Linux system, you will need to set a kernel arguments to
-disable Intel P-states. If your Linux bootloader is Grub, you can follow these
-steps:
+## Step 2 (Linux only): kernel and OS setup
 
-  * Edit /etc/default/grub (e.g. `sudo gedit /etc/default/grub`)
-  * Add `intel_pstate=disable` to `GRUB_CMDLINE_LINUX_DEFAULT`
-  * Run `sudo update-grub`
+The kernel must also be
+configured to be tickless on all CPU cores except the boot core.
+\laurie{shouldn't we tell people how to do this?}
 
-You can disable Krun's P-state check with `--disable-pstate-check`, however
-this is strongly discouraged for real benchmarking.
+### P-states
 
-### The Krun Linux Kernel
+Benchmarking is at its most accurate when Intel p-states are disabled in
+the kernel. If you are using Grub, this can be achieved as follows:
```

Comment:
I'm not even sure that is true. I'd not even mention it.

Reply:
What's not true? This hasn't really changed in a meaningful way from the old thing.

Reply:
That "Benchmarking is at its most accurate when Intel p-states are disabled".

I'm not suggesting this is your fault. I guess I'm finding new bugs.